src/Router/Router.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Application\Router;
+
+class Router
+{
+    /**
+     * Disable the constructor to simulate a static class
+     */
+    private function __construct()
+    {}
+    
+    /**
+     * Analyze the url and return the controller name, the method to call and the parameters
+     *
+     * @return Route
+     */
+    public static function run(): Route
+    {
+        // Default route : Home
+        $controller = 'Controller\BlogController';
+        $method = 'showAllPosts';
+        $params = [];
+
+        if (isset($_GET['page'])) {
+            $page = $_GET['page'];
+
+            // Blog
+            if ($page === 'blog') {
+                $controller = 'Controller\BlogController';
+                $method = 'showAllPosts';
+                $params = [];
+
+            // Blog Post
+            } elseif ($page === 'blog-post' &&
+                isset($_GET['post-id']) &&
+                is_numeric($_GET['post-id'])) {
+
+                $controller = 'Controller\BlogController';
+                $method = 'showASinglePost';
+                $params = ['postId' => $_GET['post-id']];
+
+            // Blog Admin
+            } elseif ($page === 'blog-admin') {
+                if (isset($_POST['add-post'])) {
```

Comment:
Why are you using $_POST for all your admin routing? Why not something like /admin/post?action=add_post
Or
/admin/post/add?